packages/newspringchurchapp/src/index.js

```diff
@@ -22,12 +22,22 @@ import Prayer from './prayer';
 import LandingScreen from './LandingScreen';
 import UserWebBrowser from './user-web-browser';
 import Onboarding from './ui/Onboarding';
+import NSIcon from './ui/NSIcon';
 
 const AppStatusBar = withTheme(({ theme }) => ({
   barStyle: 'dark-content',
   backgroundColor: theme.colors.paper,
 }))(StatusBar);
 
+const BrandIcon = withTheme(({ theme }) => ({
```

Comment:
I'm confused why these changes are needed? There's no match to this in core and their icons seem to work fine. In my testing this is obviously controlling stuff because the icons on the onboarding slides seem to only change size based on the size set here and not the size that is already set in the slide file. I'm not sure why we would need to override that.

Reply:
@richarddubay for this one specifically, it _has_ to be defined here to be passed in for the SMS Login screens. In Core, this is set in the `ui-auth` package which we cannot edit.